package.json

```diff
@@ -15,9 +15,9 @@
   "scripts": {
     "postinstall": "npx run-s lerna:bootstrap lerna:install lerna:install:wasmfs lerna:link",
     "build": "npm run lerna:build",
-    "dev": "npm run lerna:build:wasm-transformer && npx parcel examples/wasm-shell/index.html",
+    "cli": "./packages/cli/bin/run",
+    "dev": "npm run lerna:build:wasm-transformer && cd examples/wasm-shell && parcel index.html",
```

Comment:
Can you use [`npm-run-all`](https://www.npmjs.com/package/npm-run-all) here? The reason why I use that is because Windows is actually very popular for modern Web Dev, and && won't work on windows. 😄

Reply:
Can't use npm-run-all here because cd doesn't work with it :(